src/streamlink/plugins/cinergroup.py

```diff
@@ -22,14 +23,42 @@
         |
         haberturk\.(?:com|tv)(?:/tv)?/canliyayin
         |
-        showmax\.com\.tr/canliyayin
+        bloomberght.com/tv
         |
-        showturk\.com\.tr/canli-yayin/showturk
+        showmax\.com\.tr/(canli-yayin|canliyayin)
+        |
+        showturk\.com\.tr/(canli-yayin|canliyayin)(/showtv)?
     )/?
 """, re.VERBOSE))
 class CinerGroup(Plugin):
-    def _get_streams(self):
-        stream_url = self.session.http.get(self.url, schema=validate.Schema(
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._page = None
+
+    @property
+    def page(self):
+        if self._page is None:
+            self._page = self.session.http.get(self.url, schema=validate.Schema(
+                validate.parse_html(),
+            ))
+        return self._page
```

Comment:
This is unnecessary. The `page` property only gets accessed once and a cached HTML parsing result could easily be passed as an argument to different class methods for extracting streams. See my other comment / suggestion.